src/components/Wordbook.scss

```
@@ -0,0 +1,122 @@
+.WordbookContainer {
+    overflow: hidden;
+    background-color: #fff;
+    width: 393px;
+    height: 844px;
+    position: absolute;
+    left: 0%;
+    top: 0%;
+
+    @media screen and (max-width: 768px) {
+        width: 100%;
+    }
+
+    .WordbookHeader {
+        display: flex;
+        width: 393px;
+        height: 66px;
+        justify-content: space-between;
+        align-items: center;
+        background-color: #f2f2f7;
+
+        .ServiceName {
+            font-family: Pretendard;
+            font-size: 26px;
+            font-style: normal;
+            font-weight: 800;
+            line-height: 38px;
+            position: absolute;
+            padding: 12px 20px 16px 20px;
+            left: 0px;
+            right: 0px;
+            justify-content: space-between;
+            align-items: center;
+            float: left;
+
+            .ProfileIcon {
+                width: 36px;
+                height: 36px;
+                flex-shrink: 0;
+                float: right;
+            }
+        }
+    }
+
+    .Space {
+        width: 393px;
+        height: 32px;
+    }
+
+    .PageTitle {
+        color: #44da92;
+        text-align: center;
+        font-family: Pretendard;
+        font-size: 26px;
+        font-style: normal;
+        font-weight: 800;
+        line-height: 38px;
+        display: flex;
+        padding: 0px 20px;
+        height: 44px;
+        flex: 1 0 0;
+    }
+
+    .DateSection {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 4px;
+
+        .DateItemContainer {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 4px;
+
+            .DateItem {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                width: 353px;
+                padding: 12px 20px;
+
+                .DateInfo {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    align-items: flex-start;
+                    gap: 4px;
+                    flex: 1;
+
+                    .DateText {
+                        color: #1c1c1e;
+                        font-family: Pretendard;
```

font-family, font-size, font-style, font-weight, line-height 를 직접 지정하시는 것보다 TextStyle에 있는 title02-bold를 불러오는 것을 권장드립니다.
```suggestion
        @include title02-heavy;
```